examples/camera-app/linux/include/clusters/webrtc-provider/webrtc-provider-manager.h

```diff
@@ -69,6 +72,16 @@ class WebRTCProviderManager : public chip::app::Clusters::WebRTCTransportProvide
                                    const chip::Optional<chip::app::DataModel::Nullable<uint16_t>> & videoStreamId,
                                    const chip::Optional<chip::app::DataModel::Nullable<uint16_t>> & audioStreamId) override;
 
+    CHIP_ERROR ValidateVideoStreamID(uint16_t videoStreamId) override;
```

Comment:
Would recommend defining these functions in the CameraAVStreamManager which is the AVSM Delegate. Then we can use the CameraDevice->GetAVStreamDelegate() to call these methods. Ideally, we should have the AVSMDelegate only call CameraAVStreamMgmtServer SDK functions.

Reply:
But WebRTCProviderManager can only call its own delegate APIs, not the Delegate API from another server

Reply:
Done